.gitignore

```diff
@@ -7,6 +7,7 @@
 /spec/reports/
 /tmp/
 /Gemfile.lock
+*.swp
```

Comment:
I think a better approach is to configure this on your local machine: https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore